CHANGELOG.md

```diff
@@ -1,4 +1,28 @@
-## [2.3.2](https://github.com/commercetools/merchant-center-application-kit/compare/v2.3.2...v2.3.1) (2018-11-22)
+## [2.4.0](https://github.com/commercetools/merchant-center-application-kit/compare/v2.3.2...v2.4.0) (2018-11-26)
+
+#### 🔮 Type: Chore
+
+- Other
+  - [#137](https://github.com/commercetools/merchant-center-application-kit/pull/137) chore: publish to next dist-tag by default, add command to move to latest ([@emmenko](https://github.com/emmenko))
+- `application-shell`, `jest-preset-mc-app`
+  - [#136](https://github.com/commercetools/merchant-center-application-kit/pull/136) chore: update deps ([@emmenko](https://github.com/emmenko))
+- `i18n`
+  - [#124](https://github.com/commercetools/merchant-center-application-kit/pull/124) chore: sync en.json -> core.json after pulling messages from tx ([@emmenko](https://github.com/emmenko))
+  - [#125](https://github.com/commercetools/merchant-center-application-kit/pull/125) chore(i18n): pull translations from transifex ([@adnasa](https://github.com/adnasa))
+
+#### ✍️ Type: Documentation
+
+- `actions-global`, `application-shell-connectors`, `application-shell`, `assets`, `babel-preset-mc-app`, `browser-history`, `constants`, `eslint-config-mc-app`, `i18n`, `jest-preset-mc-app`, `l10n`, `mc-html-template`, `mc-http-server`, `mc-scripts`, `notifications`, `permissions`, `react-notifications`, `sdk`, `sentry`, `storage`, `url-utils`
+  - [#128](https://github.com/commercetools/merchant-center-application-kit/pull/128) feat: add badges. ([@montezume](https://github.com/montezume))
```

Comment:
maybe we can add semvar unrelated like we have in ui-kit, because I wouldn't bother adding this to the changelog next time 😇

Reply:
We call it: "Typed: Malcolmed".